### DISCUSSION TITLE
Restrict the backport command to a specific group of users?

### DISCUSSION THREAD

**Discussion by guits** [Mergifyio/mergify#4155] [Q&A] [answered]
Hi all!

Perhaps I've missed that detail in the documentation but I couldn't figure out whether there's a way to restrict/disable the command `@Mergifyio backport <branch name> <branch name 2>`? 

Thanks!

**Comment by sileht** [comment, marked as answer]
Currently, you can only restrict the base branch, not the destination. eg: https://docs.mergify.com/configuration/#commands-restrictions

**Comment by guits** [comment]
Thanks for your reply @sileht 

Would it be something doable? If yes, any chance this feature could show up in your roadmap? 😁 
Thanks!

**Reply by sileht** [reply]
If you want to control the destination branch, you should forbid the command entirely and use a `pull_request_rules` with action `backport` based of labels.

**Reply by guits** [reply]
sorry for asking @sileht, I might have missed it but I couldn't find how to forbid the command entirely in the documentation?
I found this https://docs.mergify.com/configuration/#commands-restrictions but I don't think this is what I'm looking for.
Any idea how to achieve that?
Thanks!

**Reply by sileht** [reply]
Just set a condition that will never match, like
`number<0`

**Reply by guits** [reply]
thanks @sileht !
